plugins/parsers/grok/influx-patterns

```diff
@@ -0,0 +1,73 @@
+# Captures are a slightly modified version of logstash "grok" patterns, with
```

Comment:
I feel like I should know this, but... any idea why we need this file when we have influx_patterns.go?

Reply:
I wasn't sure if it was something relevant.  I was under the impression that it was simply the text from influx_patterns.go.  I think it can be removed.

Reply:
Go ahead and remove this.